.github/workflows/export_to_csv.yml

```diff
@@ -129,6 +152,9 @@ jobs:
   store-csv:
     needs: [ export-to-csv ]
     runs-on: ubuntu-latest
+    # Don't upload to Google Cloud if it was triggered by hand.
+    # In that case the resulting sources.csv can be found in the build artifacts.
+    if: ${{ github.event_name != 'workflow_dispatch' }}
```

Comment:
question: There is no harm uploading the artifact to Google cloud. Why prevent it?

Reply:
I added the workflow_dispatch as a way to test the creation of the csv. I don't feel comfortable uploading it each time someone presses the button. But I could remove the restriction if you feel it's ok.